.github/workflows/tests-rs-package.yml

```diff
@@ -50,6 +54,11 @@ jobs:
           SCCACHE_REGION: ${{ vars.AWS_REGION }}
           SCCACHE_S3_KEY_PREFIX: ${{ runner.os }}/sccache/${{ steps.arch.outputs.arch }}/linux-gnu
 
+      - name: Find unused dependencies
```

Comment:
Let's move it seprate job. You don't need rust and it should very quick so just use free runner

Reply:
You do  need rust. It builds cargo-machete from sources.

Reply:
Ok, but still